### DISCUSSION TITLE
How can I create editable columns in a table, similar to text fields?

### DISCUSSION THREAD

**Discussion by asifka-ramz** [KevinVandy/material-react-table#895] [Q&A]
Hi All,

How can I create editable columns in a table, similar to text fields?

Your help would be greatly appreciated.

thanks

**Comment by dangkhoa99** [comment]
You need to set the enableEditing table option to true. Link: https://www.material-react-table.com/docs/guides/editing

**Comment by asifka-ramz** [comment]
@dangkhoa99 

Thanks for the reply. While enableEditing does make the table editable, my actual requirement is to have custom styling for editable cells or display a text box within the editable cell.

**Reply by dangkhoa99** [reply]
You can custom style with ```muiEditTextFieldProps```.
